R/getItems.R

```diff
@@ -2,13 +2,15 @@
 #'
 #' @param domainId An item domain id
 #' @param itemStatus Item status (in numeric or character),
-#                   it is possible to enter severl states like this
-#                   for example: c(1, 2, 3).
+#'                   it is possible to enter severl states like this
```

Comment:
`several`?

Reply:
yes you are correct, it should be several